code/game/gamemodes/miniantags/umbra/phantasmal_ashes.dm

```diff
@@ -0,0 +1,44 @@
+#define UMBRA_ASHES_REFORM_TIME 600 //In deciseconds, how long ashes take to reform
+
+/obj/item/phantasmal_ashes
+	name = "phantasmal ashes"
+	desc = "A shimmering pile of blue ashes with a glowing purple core."
+	icon = 'icons/obj/magic.dmi'
+	icon_state = "phantasmal_ashes"
+	w_class = 2
+	gender = PLURAL
+	origin_tech = "materials=6;bluespace=4;biotech=6" //Good origin tech if you think you have enough time to get them to research
+	var/umbra_key //The key of the umbra that these ashes came from
+	var/umbra_vitae //The vitae of the umbra that these ashes came from
+
+/obj/item/phantasmal_ashes/New()
+	..()
+	spawn(UMBRA_ASHES_REFORM_TIME)
```

Comment:
In this case, the addtimer() syntax would be (I think):
`addtimer(src, "reform", UMBRA_ASHES_REFORM_TIME)`

The addtimer takes care of the case when src is deleted. For general description, you can take a look at https://github.com/tgstation/tgstation/blob/master/code/controllers/subsystem/timer.dm#L53

Instead of spawn(), it defines your code that you want to run delayed as an "event" (think of it as a random event that occurs in-game or the running of process(). Here, you define the thing to call (usually src), the proc (from src) to call, the time when it needs to be called (same number as you'd use in spawn()).